code/modules/mob/dead/new_player/sprite_accessories.dm

```diff
@@ -309,6 +309,16 @@
 	name = "Devil Lock"
 	icon_state = "hair_devilock"
 
+/datum/sprite_accessory/hair/dolorosa
```

Comment:
Please move all these changes into a new file in the Orbstation folder somewhere.

Reply:
Also, why was icons/mob/species/human/human_face.dmi modified?

Reply:
> Please move all these changes into a new file in the Orbstation folder somewhere.

Got it.

> Also, why was icons/mob/species/human/human_face.dmi modified?

Because the only way I could think to do this was to use the original commit (which added them to human_face.dmi) and then moving them out. The actual contents shouldn't be changed, but unfortunately it's still considered "changed" by Github. Now that I've got the hairs in a separate file, I could make a new branch where that file is unchanged.